### DISCUSSION TITLE
Review process

### DISCUSSION THREAD

**Discussion by minna-xD** [pik-gane/vodle#213] [General]
I thought I'd move the discussion about translation tool and reviewing from my PR to here.

I'd be quite interested in trying out the review feature in Crowdin, actually. The tool is nice and efficient for translation with the translation memory and commenting function, and so it would be a good tool to review the translations as well. It clearly marks texts that are ready for reviewing and those that are already accepted. It also acts as a project management tool.

Translation memory is very useful in both one-translator and multiple-translator setting, but **_especially_** in a multiple-translator setting when people are working on the translation more or less simultaneously. 

You could even utilize Crowdin's terminology tool, if you want to get serious. 🙂 Because vodle has those special concepts, you might want to add them as a glossary. That way they would appear in the sidebar for translators, a translation could be added when the term first appears and it would be more certain that the term would be consistent in subsequent strings. Translation memory is a good source for terms, too, but translators need to remember to search specifically for any new concepts they come across because only some of the existing translations appear during translation. This concerns mainly teams of translators. With one translator, they are usually able to keep in mind the concepts and their own translation decisions.

On _acquiring_ reviewers (and translators), I don't know because I assume this would stay a volunteer effort. 🤔 Perhaps this app would interest e.g. math (or other subject) students and enthusiasts so you might be able to get eager volunteers from such groups (a suitable subreddit..?).

With "amateur translators" it would be good to have someone in the reviewers who has a knack for grammar rules and a good eye for spotting mistakes but that is just my personal opinion (I'm a nitpicker). 😇

On the process, in my work the reviewing was done so that the reviewer(s) would give comments and suggestions but the translator(s) would have the final say (this is because they have done the mental work for the specific source material and know the reasons behind their translation decisions). Crowdin handles this quite nicely, because reviewers can add fixes directly in the reviewing mode, they will show up as alternate translations and then e.g. the original translator could go through these and approve. (There also seems to be a _voting_ feature that could be used to find the best translation! 😉)

I don't know if the above was of any help or just confusing, but if there are any questions, I'll be happy to answer.

**Comment by mensch72** [comment]
Dear @minna-xD , this is extremely helpful! Thank you so much for taking the time to review Crowdin's features. I will now...

- test the review features with a few reviewers
- include some of your text into CONTRIBUTING.md and into comment blocks in en.json
- try adding crucial terminology to Crowdin's terminology tool to guide future translators
